opendevin/action/browse.py

```diff
@@ -14,6 +16,16 @@
     from opendevin.controller import AgentController
 
 
+def browser_env_step(action_str: str, conn: Connection):
+    unique_request_id = str(uuid.uuid4())
+    conn.send((unique_request_id, {'action': action_str}))
+    while True:
```

Comment:
Will this spin at 100% CPU? If so, it'd be good to either find another alternative or open an issue noting that this should be fixed.

Reply:
even a `sleep(.1)` in here might help